### PR TITLE
Update Neovim.ts

### PR DESCRIPTION
**Pull request by KadoBOT**
Add `style` to OpenWindowOptions

### CODE REVIEW

File: packages/neovim/src/api/Neovim.ts [hunk 1]
```diff
@@ -100,6 +100,9 @@ export interface OpenWindowOptions {
 
   // height of the window
   height: number;
+  
+  // Currently only takes one non-empty value: "minimal". Nvim will display the window with many UI options disabled
+  style?: 'minimal'
```
Comment (rhysd) [inline packages/neovim/src/api/Neovim.ts:105]: `;` is missing at end of line

**Review by rhysd — CHANGES_REQUESTED** [review]

**Comment by rhysd** [comment]
@KadoBOT Can you show a link to document of the `style` option in Neovim?

**Review by rhysd — APPROVED** [review]

**Comment by rhysd** [comment]
https://dev.azure.com/neovim/node-client/_build/results?buildId=216&view=logs&j=775b36c7-0acf-56fe-5683-ee983d10b23d&t=db2cdf11-2dd7-537a-2325-1e72eed53a44&l=28

Can you resolve the prettier warning?

```
/Users/runner/work/1/s/packages/neovim/src/api/Neovim.ts
  103:1  error  Delete `··`  prettier/prettier

✖ 1 problem (1 error, 0 warnings)
  1 error and 0 warnings potentially fixable with the `--fix` option.
```

**Comment by rhysd** [comment]
Yeah, I believe they are valid.

**Comment by rhysd** [comment]
@billyvg @justinmk Can we make a new release?

**Comment by billyvg** [comment]
4.10.1 published -- thoughts on moving to GitHub Actions? Can move add a publish workflow when we make a GitHub Release too.

**Comment by rhysd** [comment]
> thoughts on moving to GitHub Actions? 

That would be nice idea though I'm not understanding how to manage npm credentials and 2FA on GHA.

**Pull request merged**